packages/server/lambdas/alfresco/searchQueryBuilder/index.ts

```diff
@@ -11,7 +11,7 @@ export const searchQueryBuilder = ({
 }: QueryRequest): Query => {
   switch (language) {
     case QueryLanguage.LUCENE:
-      const luceneQueryBuilder = new LuceneQueryBuilder();
+      const luceneQueryBuilder = new LuceneQueryBuilder('/app:company_home/st:sites/cm:ratat-extra//*');
```

Comment:
Does this work in both dev and prod?

Reply:
(Accidental approval when I just meant to comment. Revoked my approval for now)

Reply:
Oh true. prod has different site name than dev. Is it good idea to store path name in parameter store?

Reply:
I'd save it there, yes.